tex/iii.1/_0083c.tex

```diff
@@ -30,14 +31,13 @@
 Цей випадок не дає нових точок зору і розв’язується за допомогою
 загальної формули, даної в рубриці: II. $m'$ змінюється.
 
-\pfbreak{}
```

Comment:
We can't remove the \pfbreak (see above) but can replace it with plainnbreak

Reply:
Understood. I have reverted back to \pfbreak